live-examples/html-examples/meta.json

```diff
@@ -1,13 +1,21 @@
 {
     "pages": {
+        "abbr": {
+            "baseTmpl": "tmpl/live-tabbed-tmpl.html",
+            "exampleCode": "live-examples/html-examples/abbr.html",
+            "cssExampleSrc": "live-examples/html-examples/css/abbr.css",
+            "fileName": "abbr.html",
+            "title": "HTML Demo: &lt;abbr&gt;",
```

Comment:
Ah, so I see it is a problem in the title. This will resolve that problem - https://github.com/mdn/bob/issues/18